Selenium.Axe/Selenium.Axe/AxeBuilder.cs

```diff
@@ -42,38 +109,31 @@ public AxeBuilder(IWebDriver webDriver, AxeBuilderOptions options)
             _webDriver.Inject(options.ScriptProvider);
         }
 
-        /// <summary>
-        /// Execute the script into the target.
-        /// </summary>
-        /// <param name="command">Script to execute.</param>
-        /// <param name="args"></param>
-        private AxeResult Execute(string command, params object[] args)
-        {
-            string stringifiedResult = (string) ((IJavaScriptExecutor)_webDriver).ExecuteAsyncScript(command, args);
-            var jObject = JObject.Parse(stringifiedResult);
-            return new AxeResult(jObject);   
-        }
-
         /// <summary>
         /// Selectors to include in the validation.
         /// </summary>
         /// <param name="selectors">Any valid CSS selectors</param>
         /// <returns></returns>
         public AxeBuilder Include(params string[] selectors)
```

Comment:
It would be nice to clarify in the param name/comment that the intended usage is `.Include("#some-iframe", "#some-nested-iframe", "#element-in-nested-iframe")`, *not* `.Include("#some-element-in-root-frame", "#some-other-element-in-root-frame")` - I think this would be surprising for someone not deeply familiar with axe-core (I had to look up what the original `IncludeExcludeManager` behavior was to feel confident this wasn't changing it)